CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Bugfix: Fixed being unable to see the usercard of VIPs who have Asian language display names. (#4174)
 - Bugfix: Fixed the wrong right-click menu showing in the chat input box. (#4177)
+- Bugfix: Do not parent popup windows on Windows OS. (#4181)
```

Comment:
```suggestion
- Bugfix: Fixed popup windows not appearing/minimizing correctly on the Windows taskbar. (#4181)
```
more friendly to the user, I can apply before merge after this is tested by another windows user later.

Reply:
I agree, this is way more accessible. Feel free to apply whenever you see fit :+1: